### PR TITLE
Documented automated global_operations

### PR DESCRIPTION
**Pull request by Toflar**
Implements https://github.com/contao/docs/issues/1300.

### CODE REVIEW

**Review by fritzmg — COMMENTED** [review]
Do you want to add a `version-tag` in the notices here too?

File: docs/dev/reference/dca/list.md [hunk 1]
```diff
@@ -128,13 +142,13 @@ $GLOBALS['TL_DCA']['tl_example']['list']['operations'] = [
 | route           | Symfony Route Name (`string`)     | {{< version-tag "4.7" >}} The button will redirect to the given Symfony route.                                                               |
 
 {{% notice "note" %}}
-Since Contao **5.0** you do not have to define any settings for standard operations anymore. Instead you can give a list
+Since Contao **5.0** you do not have to define any settings for standard operations anymore. Instead, you can give a list
```
Comment (fritzmg) [inline docs/dev/reference/dca/list.md]:
```suggestion
{{< version-tag "5.3" >}} You do not have to define any settings for standard operations anymore. Instead, you can give a list
```

Comment (Toflar) [reply docs/dev/reference/dca/list.md]: That propsal is wrong, that's an upstream conflict. Resolved.

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @Toflar :)

**Pull request merged**